crates/egui/src/input_state.rs

```diff
@@ -794,6 +859,9 @@ pub struct PointerState {
 
     /// All button events that occurred this frame
     pub(crate) pointer_events: Vec<PointerEvent>,
+
+    /// Input state management configuration.
+    input_options: InputOptions,
```

Comment:
We should note here that this is copied from `egui::Options` at the start of each frame, and is only here for convenience